pkg/integrations/loki/loki.go

```diff
@@ -56,6 +56,7 @@ const (
 	DefaultLimit       = 100
 	MsToNanoConversion = 1_000_000
 	DAY_IN_MS          = 86400000
+	HOUR_IN_MS         = 3600000
```

Comment:
use `_` as the thousands seperator for this and the above day_in_ms, makes code reading easier